JSDemos/Demos/PivotGrid/SummaryDisplayModes/description.md

```diff
@@ -1 +1,19 @@
-The PivotGrid component supports several modes for summary calculation, such as&nbsp;a&nbsp;variation or&nbsp;percent of&nbsp;a&nbsp;total value. Select the required mode from the context menu appearing when you right-click the &laquo;Relative Sales&raquo; field.
\ No newline at end of file
+Summaries are values displayed in pivot grid cells. You can configure summaries in the [fields](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/) that belong to the *"data"* area (`Relative Sales` in this demo). 
```

Comment:
I presume this demo is advanced enough so this part can be omitted:
```suggestion
To process already calculated summary values, you can apply a predefined or your custom function. This allows you to take into account neighboring cells' values.
```
Should PivotGrid have at least one field whose `area` is set to "data"?